features/topology-map/org.opennms.features.topology.app/src/main/java/org/opennms/features/topology/app/internal/ui/InfoPanel.java

```diff
@@ -70,6 +73,9 @@ public InfoPanel(SearchBox searchBox) {
 
         staticComponents.add(searchBox);
         staticComponents.add(toggleButton);
+
+        graphRenderContainer = new InlineGraphContainer();
```

Comment:
It's added as a component in the line below. The simple existence of the component is enough to trigger a render call on each update.